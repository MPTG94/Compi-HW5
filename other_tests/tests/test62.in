void Func(bool s, bool t) {
	print("If you believe it will work out, you’ll see opportunities. If you believe it won’t, you will see obstacles. – Wayne Dyer");
}
void maxTestsTestPassRelopToFunc() {
	Func(1 < 2, 3 < 4);
}

void main() {
     maxTestsTestPassRelopToFunc();
}

