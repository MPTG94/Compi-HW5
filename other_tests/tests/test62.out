If you believe it will work out, you’ll see opportunities. If you believe it won’t, you will see obstacles. – Wayne Dyer
